assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -86,6 +91,38 @@ function MyStories() {
     });
   }, [stories, typeaheadValue]);
 
+  const setPageSize = useCallback(() => {
+    const { innerWidth } = window;
+
+    if (innerWidth <= theme.breakpoint.raw.min) {
+      _setPageSize({
+        width: theme.grid.min.itemWidth,
```

Comment:
Not convinced that this belongs in the theme. But even if it does - we should probably only record width there and use `PAGE_WIDTH/PAGE_HEIGHT` constants to construct the correct aspect ratio. We're trying to limit the number of places where aspect ratio information is stored to 1.

Reply:
I think that makes sense. The reason for having both is the math can cause floats, now we can use floor to prevent that.